content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Multicluster Deployment Considerations
+description: Describes considerations when configuring your Multicluster Istio deployment.
+weight: 20
+keywords:
+  - multiple-clusters
+  - control-plane
+  - networks
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Before you install Istio on {{< gloss "multicluster" >}}multiple clusters{{< /gloss >}}, there are some things to
+consider regarding the requirements of your system which may impact your installation choices.
+
+## Control Planes
+
+This section describes things you’ll need to consider when deploying an Istio {{< gloss >}}control plane{{< /gloss >}}
+in your environment.
+
+### Configuration
+
+An Istio control plane in a {{< gloss >}}primary cluster{{< /gloss >}} receives its configuration
+(i.e. `Service`/`ServiceEntry`, `DestinationRule`, etc.) from the Kubernetes API Server residing in the same
+{{< gloss >}}cluster{{< /gloss >}}.
+
+When a mesh contains multiple primary clusters, each control plane has an independent source of configuration.
+
+{{< image width="80%"
+    link="configuration.svg"
+    caption="Primary clusters with independent sources of configuration"
+    >}}
+
+This model has the following advantages:
+
+* Reduces the blast radius of bad configuration and helps to maintain high availability for the control plane.
+* Allows administrators more fine-grained control over configuration rollout (e.g. one cluster at a time). For large
+meshes with many clusters, this process can be automated with CI/CD pipelines.
+* Allows selective exposure of `Service`/`ServiceEntry` across cluster boundaries. For example, an administrator may
+choose to deploy the HelloWorld service to Cluster A, but not Cluster B. This is an advanced use case, but may be
+desirable for stronger isolation between clusters.
+
+This duplication of configuration across primary clusters can lead to issues, however. In particular, Kubernetes
```

Comment:
"can lead to issues" - is this too strong?  seems a red flag but it is not.   I think it is more like:
This duplication of configuration across primary clusters needs additional steup, either by deploying the service to the consuming cluster or create service entry on the consuming cluster.